Documentation/kvstore.rst

```diff
@@ -0,0 +1,70 @@
+Key-Value Store
+###############
+
+Cilium uses an external key-value store to exchange information across multiple
+Cilium instances:
+
+Layout
+======
+
+All data is stored under a common key prefix:
+
+===================== ====================
+Prefix                Description
+===================== ====================
+``cilium/``           All keys share this common prefix.
+``cilium/state/``     State stored by agents, data is automatically recreated on removal or corruption.
+``cilium/conf/``      Configuration, agents will read this to retrieve
+                      configuration but will never write to these keys.
+                      Used to store agent configuration in the kvstore.
```

Comment:
For me, this sentence does not seems right. With the previous sentence you say all "agents will read" but here you say "used to store agent configuration".

If the agents, AFAIK, are the only client to the kvstore, who's going to store that agent configuration?

Reply:
Given we are not using `/conf/` yet, I've removed this from the doc.

"stored" does not mean that that the agent writes to it. It could be another tool leaving behind configuration for all agents.